command/build.py

```diff
@@ -1,8 +1,37 @@
+import yaml as pyYaml
+import command.identify as identify
+
 class BuildCmd():
     name = 'build'
 
-    def __init__(self):	
+    def __init__(self):
```

Comment:
empty `__init__` can be omitted

Reply:
@MartinBasti addressed in latest commit.